src/Mvc/Mvc.Testing/src/Microsoft.AspNetCore.Mvc.Testing.targets

```diff
@@ -25,28 +30,23 @@
 
   <Target Name="_AddContentRootForProjectReferences" BeforeTargets="GetAssemblyAttributes" DependsOnTargets="_ResolveMvcTestProjectReferences">
     <ItemGroup>
-      <WebApplicationFactoryContentRootAttribute
-        Condition="'%(_ContentRootProjectReferences.Identity)' != ''"
-        Include="%(_ContentRootProjectReferences.Identity)"
-        AssemblyName="%(_ContentRootProjectReferences.FusionName)"
-        ContentRootPath="$([System.IO.Path]::GetDirectoryName(%(_ContentRootProjectReferences.MSBuildSourceProjectFile)))"
-        ContentRootTest="$([System.IO.Path]::GetFileName(%(_ContentRootProjectReferences.MSBuildSourceProjectFile)))"
-        Priority="0" />
+      <_ManifestProjects Include="%(_ContentRootProjectReferences.FusionName)">
+        <ContentRoot>$([System.IO.Path]::GetDirectoryName(%(_ContentRootProjectReferences.MSBuildSourceProjectFile)))</ContentRoot>
+      </_ManifestProjects>
     </ItemGroup>
 
+    <GenerateMvcTestManifestTask ManifestPath="$(IntermediateOutputPath)AppManifest.json" Projects="@(_ManifestProjects)"/>
+
     <ItemGroup>
-      <AssemblyAttribute
-        Condition=" '%(WebApplicationFactoryContentRootAttribute.Identity)' != '' "
-        Include="Microsoft.AspNetCore.Mvc.Testing.WebApplicationFactoryContentRootAttribute">
-        <_Parameter1>%(WebApplicationFactoryContentRootAttribute.AssemblyName)</_Parameter1>
-        <_Parameter2>%(WebApplicationFactoryContentRootAttribute.ContentRootPath)</_Parameter2>
-        <_Parameter3>%(WebApplicationFactoryContentRootAttribute.ContentRootTest)</_Parameter3>
-        <_Parameter4>%(WebApplicationFactoryContentRootAttribute.Priority)</_Parameter4>
-      </AssemblyAttribute>
+      <ContentWithTargetPath Include="$(IntermediateOutputPath)AppManifest.json"
```

Comment:
We should probably pick a better name before merging...  MvcTestingManifest.json?

Reply:
Updated it to `MvcTestingAppManifest.json`.